README.md

```diff
@@ -116,7 +116,7 @@ Realm Flutter package is published to [realm](https://pub.dev/packages/realm).
 * Add `realm` package to a Flutter application.
 
     ```
-    flutter pub add realm
+    dart pub add realm
```

Comment:
```suggestion
    flutter pub add realm
```
We discussed to leave `flutter pub add` in the docs and readme files until the command is deprecated.